### ISSUE TITLE
Release notes: NVDA accessibility improvement to date picker

### ISSUE THREAD

**Issue by helennickols** [ministryofjustice/moj-frontend#1802]
## What

Send a Slack post about the accessibility improvements to the date picker. 

## Why

Help people know that this has been completed.

## Who needs to be involved

- Interaction Designer
- Content Designer
- Developer

## Done

Post is written and released.
